### PR TITLE
feat(pie-form-label): DSW-1585 update pie-form-label to work with form components

### PR DESCRIPTION
**Pull request by fernandofranca**
Updates Next and Nuxt sample apps `form` page to use the `pie-form-label` component.

Note: it should not be merged yet, as it is using the snapshot package versions.

### CODE REVIEW

File: nextjs-app/src/pages/form.tsx [hunk 1]
```diff
@@ -67,42 +68,47 @@ export default function Form() {
                     onInput={handleUsernameInput as any}
                     type="text"></PieInput>
 
-                <label htmlFor="email">
+                <PieFormLabel for="email">
                     Email:
-                </label>
-                <input
+                </PieFormLabel>
+                <PieInput
```
Comment (jamieomaguire) [inline nextjs-app/src/pages/form.tsx:74]: Please could we leave these input fields as native HTML? Only the first one should be using `PieInput` for now. 

I have a separate ticket to integrate the remaining types when they are ready to test (they aren't just yet)

File: nextjs-app/src/pages/form.tsx [hunk 2]
```diff
@@ -67,42 +68,47 @@ export default function Form() {
...
-                    onChange={handlePasswordConfirmationChange}
-                    type="password" />
+                    value={passwordConfirmation}
+                    onInput={handlePasswordConfirmationChange as any}
+                    type="password"></PieInput>
 
                 <div className="form-controls">
+                    <PieFormLabel for="approveSettings">
+                        Approve settings
+                    </PieFormLabel>
```
Comment (jamieomaguire) [inline nextjs-app/src/pages/form.tsx:110]: I am clicking the label for the switch on the deployed NextJS app but it does not seem to visually focus or 'check' the switch

Comment (fernandofranca) [reply nextjs-app/src/pages/form.tsx:110]: Can you try removing the other Switch component from the DOM and trying clicking again?

Comment (fernandofranca) [reply nextjs-app/src/pages/form.tsx:110]: When I was testing the changes I had one component of each (input and switch) and they didn't had that issue. 🤔

Comment (jamieomaguire) [reply nextjs-app/src/pages/form.tsx:110]: Ah I think you've got it - removing the second switch appears to resolve the issue when I test the branch locally. I wonder if it is to do with the duplicate ID attributes we spotted on the underlying input?

Comment (fernandofranca) [reply nextjs-app/src/pages/form.tsx:107]: I'll arrange a ticket for checking that.

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**